.github/workflows/buildlint.yml

```
@@ -16,12 +16,12 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [12.x, 14.x, 16.x]
```

```suggestion
        node-version: [16.x]
```
node 16要求してくるから古いのいらなさそう

---

それはそう